python/ray/rllib/test/test_checkpoint_restore.py

```diff
@@ -21,13 +20,14 @@ def get_mean_action(alg, obs):
 ray.init()
 
 CONFIGS = {
+    "ES": {"episodes_per_batch": 10, "timesteps_per_batch": 100},
     "DQN": {},
     "PPO": {"num_sgd_iter": 5, "timesteps_per_batch": 1000},
     "A3C": {"use_lstm": False},
 }
 
 # https://github.com/ray-project/ray/issues/1062 for enabling ES test as well
```

Comment:
we can remove this line?

Reply:
No we need it. Not sure what you mean.

Reply:
This comment is in ref to the comment L29. In the PR you mention that #1062 is addressed, so I figured that you should remove this comment referring to 1062 indicating that a test was missing...

Reply:
Got it, fixed!